crux_core/src/bridge/mod.rs

```diff
@@ -0,0 +1,83 @@
+mod registry;
+pub mod step_serde;
```

Comment:
Don't think this module needs to be public, since it only adds a public method onto an existing type.

```suggestion
mod step_serde;
```